extensions/src/Smithy.Identity.Abstractions/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Reflection;
+
+[assembly: AssemblyTitle("Smithy.Identity.Abstractions")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyProduct("Smithy identity and authentication abstractions for the .NET SDK")]
+[assembly: AssemblyCompany("Amazon.com, Inc")]
+[assembly: AssemblyCopyright("Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.")]
+[assembly: AssemblyTrademark("")]
+
+[assembly: AssemblyVersion("0.1")]
```

Comment:
Can you do the same as @boblodgett did on the extensions AssemblyInfo and add the `#if` for .NET Framework with how we handle `AssemblyVersion`? https://github.com/aws/aws-sdk-net/blob/v4-development/extensions/src/AWSSDK.Extensions.CrtIntegration/Properties/AssemblyInfo.cs#L19

This also means in the csproj you need to add the .NET Framework target 4.7.2.

Reply:
Yes, I added the `#if` statements in https://github.com/aws/aws-sdk-net/pull/3330/commits/8d996a29a697a419323c524209c5af250b4d5280 (I ended up using `0.1` for both versions for now, let me know if I should use a different value instead).